tests/runtimes/local/runtime_test.py

```diff
@@ -332,3 +333,34 @@ def testRuntimeScanList_whenDockerIsDown_DoesNotCrash(
     scans = local_runtime.LocalRuntime().list()
 
     assert scans is None
+
+
+def testCheckServicesMethod_whenServicesAreStopped_shouldExit(
+    mocker: plugin.MockerFixture, run_scan_mock: None
+) -> None:
+    """Unit test for Check services method when services are stopped"""
+    mocker.patch(
+        "ostorlab.runtimes.definitions.AgentSettings.container_image",
+        return_value="agent_42_docker_image",
+        new_callable=mocker.PropertyMock,
+    )
+    mocker.patch("ostorlab.runtimes.local.agent_runtime.AgentRuntime")
+    mocker.patch("ostorlab.runtimes.local.services.mq.LocalRabbitMQ.is_service_healthy")
+    mocker.patch("ostorlab.runtimes.local.services.redis.LocalRedis.is_service_healthy")
+    exit_mock = mocker.patch("sys.exit")
+    local_runtime_instance = local_runtime.LocalRuntime()
+    local_runtime_instance.follow = ["service1"]
+    agent_group_definition = definitions.AgentGroupDefinition(
+        agents=[definitions.AgentSettings(key="agent/ostorlab/agent42")]
+    )
+
+    local_runtime_instance.scan(
+        title="test local",
+        agent_group_definition=agent_group_definition,
+        assets=[android_apk.AndroidApk(content=b"APK")],
+    )
+    sleep(1)
```

Comment:
Why the sleep?

Reply:
to give the thread sometime to check the services and exit